helm/reana/templates/NOTES.txt

```diff
@@ -1,5 +1,13 @@
 The REANA system has been installed!
 
+If you are upgrading from a previous REANA release series, you may need to
+upgrade database schema now:
+
+   $ kubectl -n {{ .Release.Namespace }} exec -i -t deployment/{{ .Release.Name }}-server -- reana-db alembic upgrade
+
+Please see the dedicated upgrade documentation at
+<https://docs.reana.io/administration/management/migrating-db/>
```

Comment:
One more thing: Does DB migration belongs to "Managing..."? To me it is rather part of "Deploying...", since DB upgrade is a general topic thing, valid all the time for all the deployments, whilst "Managing" has specific topics listed there once the site is up and running, managing users, managing quotas, managing nodes, etc.  

I'd also probably call it "DB upgrade", not "DB migration", which might evoke a feeling of moving from PostgreSQL version N to M, or to another database solution.

Perhaps we can review that one first, settle on a URL there, and come back to this one afterwards.

Reply:
I called it "Migrating" as this is the term that [alembic](https://alembic.sqlalchemy.org/en/latest/) uses officially.

> Alembic is a lightweight database migration tool for usage with the SQLAlchemy Database Toolkit for Python.

Moved to deployment.

Reply:
I see the motivation :+1: However, since people would usually speak of "upgrading REANA", "upgrading client", "upgrading Python" etc from one version to a new version, it seems more end-user-friendly and more consistent to use the term "upgrading DB" as well. (Besides, the Alembic command itself is called `upgrade`...)